.ci/go-test.sh

```diff
@@ -108,6 +108,12 @@ test_coverage()
 			printf "INFO: Running 'go test' as %s user on package '%s' with flags '%s'\n" \
 				"$user" "$pkg" "$go_test_flags"
 
+			# Run virtcontainers tests only if user is root.
```

Comment:
Which is a little sad, but iirc it is because the vc tests need to install the cni plugins etc., which has to be done as root?
I wonder if the better fix is to ensure we `sudo` all the bits in the vc test setup so we can still run the tests?